### ISSUE TITLE
The development set in tag set extension experiment

### ISSUE THREAD

**Issue by taonlp** [asappresearch/structshot#13]
As described in the paper, the training set and test set are modified in the tag set extension experiment. But what about the development set? If it is masked the same as the training set, then the support set cannot be sampled when testing. If it is masked the same as the test set, then how to pick up checkpoints during training? I am confused with this setting, could you please give me some advice and I would be very grateful.
